Sources/Commands/SwiftPackageTool.swift

```diff
@@ -447,6 +464,8 @@ public class PackageToolOptions: ToolOptions {
         var shouldForceRemove = false
     }
 
+    var dryRun = false
```

Comment:
Maybe we should call this variable `updateDryRun`?

Reply:
done